mock/world/worldAccountMap.go

```diff
@@ -69,6 +70,10 @@ func (am AccountMap) CreateSmartContractAccountWithCodeHash(owner []byte, addres
 
 // PutAccount inserts account based on address.
 func (am AccountMap) PutAccount(account *Account) {
+	if account.Code != nil && account.CodeHash == nil {
+		hash, _ := hashing.NewHasher().Sha256(account.Code)
```

Comment:
slightly improve the code by making `hashing.NewHasher()` an unexported package variable

Reply:
Done.